sonar-zap-plugin/src/main/java/org/sonar/zaproxy/ZapPlugin.java

```diff
@@ -1,6 +1,6 @@
 /*
  * ZAP Plugin for SonarQube
- * Copyright (C) 2015 Steve Springett
+ * Copyright (C) 2015-2017 Steve Springett
```

Comment:
I love this detail-oriented change :)

Reply:
Well, this was not entirely deliberate but required by the "license-maven-plugin" to be able to build the project ;)

Reply:
I'm so gonna have a look at this plugin for my own projects!